be/src/exprs/agg/java_udaf_function.h

```diff
@@ -74,11 +76,17 @@ class JavaUDAFAggregateFunction : public AggregateFunction {
 
     void serialize_to_column(FunctionContext* ctx __attribute__((unused)), ConstAggDataPtr __restrict state,
                              Column* to) const override final {
+        BinaryColumn* column = nullptr;
         // TODO serialize
-        DCHECK(!to->is_nullable());
-        DCHECK(to->is_binary());
+        if (to->is_nullable()) {
+            auto* null_column = down_cast<NullableColumn*>(to);
+            null_column->null_column()->append_default();
```

Comment:
append_default is confusing, suggest to use append(DATUM_NULL) or append(DATUM_NOT_NULL);

Reply:
fixed